Storage/REST-API.md

```diff
@@ -0,0 +1,185 @@
+# RESTful binding for Storage, RFC
+
+This document describes the RESTful API with CRUD functionality for Current Storage.
+
+### Bindings semantics
+`TODO: Describe storage field <-> endpoint bindings.`
+
+### Response schema
+
+The response body is always either a valid JSON or empty with `204 (No content)` error code.
```

Comment:
I'd say there's no real value in 204, nobody ever is going to treat it differently from 200 with Content-Length: 0. A good client would probably shun it under a blanket [200;300) condition, so this just takes space in the spec.